docs/proposals/20230612-node-os-image-caching.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Proposal Template
+authors:
+  - "@bennycortese"
+reviewers:
+  - "@nojnhuh"
+  - "@CecileRobertMichon"
+  - "@jackfrancis"
+  - "@willie-yao"
+creation-date: 2023-06-12
+last-updated: 2023-06-12
+
+status: provisional|experimental|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - "/docs/proposals/20190101-we-heard-you-like-proposals.md"
+  - "/docs/proposals/20190102-everyone-gets-a-proposal.md"
+replaces:
+  - "/docs/proposals/20181231-replaced-proposal.md"
+superseded-by:
+  - "/docs/proposals/20190104-superceding-proposal.md"
```

Comment:
Passed through, if you have any other proposal suggestions to link here feel free to share them, I just added a reference to the prototype pattern on wikipedia and removed replaces/superseded by since they don't really fit here